src/Sentry/Internal/Hub.cs

```diff
@@ -14,6 +14,7 @@ internal class Hub : IHubEx, IDisposable
     private readonly SentryOptions _options;
     private readonly RandomValuesFactory _randomValuesFactory;
     private readonly Enricher _enricher;
+    private readonly string _sentryBaseUrl;
```

Comment:
if we get rid of the extension method, we don't need this field here either, right?